### PR TITLE
Add current time to quote

### PR DESCRIPTION
**Pull request by plondon**

### CODE REVIEW

**Review by Sjors — CHANGES_REQUESTED** [review]
This is in the frontend at the moment:
```js
let timeToExpiration = new Date(quote.expiresAt) - new Date(quote.timeOfRequest) - 1000
```
Can you add a `timeToExpiration`property to bitcoin-exchange client?

Can you also default `_timeOfRequest` to the current system time? That way it will work for all partners without having to modify them immediately.

**Comment by plondon** [comment]
Added timeToExpiration

File: src/quote.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ class Quote {
 
   get expiresAt () { return this._expiresAt; }
 
+  get timeOfRequest () { return this._timeOfRequest || new Date().getTime(); }
+
+  get timeToExpiration () { return this._timeToExpiration; }
```
Comment (Sjors) [inline src/quote.js]: `timeToExpiration` should calculated in the getter...

File: src/quote.js [hunk 2]
```diff
@@ -44,6 +44,10 @@ class Quote {
 
   get expiresAt () { return this._expiresAt; }
 
+  get timeOfRequest () { return this._timeOfRequest || new Date().getTime(); }
```
Comment (Sjors) [inline src/quote.js]: This doesn't make sense. `quote.timeOfRequest` would just return the current time all the time.

I would set `this._timeOfRequest = new Date().getTime()` in the quote constructor. That will be overridden in SFOX subclass.

**Review by Sjors — APPROVED** [review]

**Pull request merged**